python/dataproc_templates/jdbc/README.md

```diff
@@ -334,6 +335,7 @@ usage: main.py --template JDBCTOGCS \
     --jdbctogcs.input.url JDBCTOGCS.INPUT.URL \
     --jdbctogcs.input.driver JDBCTOGCS.INPUT.DRIVER \
     --jdbctogcs.input.table JDBCTOGCS.INPUT.TABLE \
+    --jdbctogcs.input.sql.query JDBCTOGCS.INPUT.SQL.QUERY \ (refer NOTE)
```

Comment:
I wasn't sure how to document this, let me know if there could be a better way!

Reply:
Lets remove either one of them from usage as they would never be used together. I would suggest adding note below for the usage for jdbctogcs.input.sql.query. ref. https://github.com/GoogleCloudPlatform/dataproc-templates/tree/main/java/src/main/java/com/google/cloud/dataproc/templates/jdbc#:~:text=username%3E%26password%3D%3Cpassword%3E%27-,Have%20SQL%20query%20within%20double%20quotes.%20Example%2C,-%2D%2DtemplateProperty%20%20%27jdbctogcs.sql